### PR TITLE
describe how to record changes without contradiction

### PR DESCRIPTION
**Pull request by mtbc**
Resolves CHRON-377 by explaining how to work with immutability.

# PR Checklist

## Please complete this checklist after opening your PR

* [x] I have updated documentation as necessary
* [x] I have updated helm chart(s) if needed
* [x] I have added tests if needed
* [x] All new and existing tests are passing
* [x] Any breaking changes are clearly flagged and documented
* [x] I’ve included a link to any relevant ticket(s)

### CODE REVIEW

File: docs/recording_provenance.md [hunk 1]
```diff
@@ -115,6 +115,33 @@ mutation {
...
+then [query the activity timeline](./querying_provenance.md#activity-timeline)
+to retrieve the history of when the person was where.
+
+Similarly, in revising the description of an artifact, each description can
+be an entity derived from the artifact. An agent can engage in the activities
+of "inserting" one of the descriptions and "removing" the other. As above, the
+history of an artifact's descriptions is available for query.
+
+Using activities in this way allows changes over time to be represented while
+the value set for every attribute remains constant. If one chooses to take
```
Comment (ryanbtp) [inline docs/recording_provenance.md:140]: probably don't need 'set'

File: docs/recording_provenance.md [hunk 2]
```diff
@@ -115,6 +115,33 @@ mutation {
 }
 ```
 
+### Representing Mutability
+
+Working through our [example domain definitions](https://examples.btp.works/),
+one may notice that some define attributes like "location" that may seem as if
+they could be mutable. They are *not* mutable: the above rules on
+contradiction hold firm. Such examples use entities or agents in fixed
+locations, and illustrate other facets of modeling domains in Chronicle.
```
Comment (ryanbtp) [inline docs/recording_provenance.md:124]: Possibly say something explicit about having to model changes to attributes as provenance?

**Review by ryanbtp — CHANGES_REQUESTED** [review]

File: docs/recording_provenance.md [hunk 3]
```diff
@@ -115,6 +115,35 @@ mutation {
...
+
+With attributes like the location of an entity or an agent, in cases where
+they may need updating, one approach is to instead model the locations as
+entities in their own right. Then, for example, an person could engage in the
```
Comment (suchapalaver) [inline docs/recording_provenance.md:128]:
```suggestion
entities in their own right. Then, for example, a person could engage in the
```

File: docs/recording_provenance.md [hunk 4]
```diff
@@ -115,6 +115,35 @@ mutation {
...
+*activity* of "entering" or "leaving" a location. By marking the activity with
+a [time](./provenance_concepts.md#start-and-end), even a duration, one may
+then [query the activity timeline](./querying_provenance.md#activity-timeline)
+to retrieve the history of when the person was where.
+
+Similarly, in revising the description of an artifact, each description can
+be an entity derived from the artifact. An agent can engage in the activities
```
Comment (suchapalaver) [inline docs/recording_provenance.md:135]:
```suggestion
be an entity derived from the artifact itself. An agent can engage in the activities
```

**Review by suchapalaver — CHANGES_REQUESTED** [review]

**Review by ryanbtp — APPROVED** [review]

**Review by suchapalaver — APPROVED** [review]

**Pull request merged**